easybuild/easyconfigs/r/rpy2/rpy2-2.8.2-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'rpy2'
+version = '2.8.2'
+
+homepage = 'http://rpy.sourceforge.net/'
+description = """rpy2 is a redesign and rewrite of rpy. It is providing a low-level 
+interface to R from Python, a proposed high-level interface, including wrappers to 
+graphical libraries, as well as R-like structures and functions. """
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2c1a313df4e64236dcfe1078ce847b8e3c180656c894928d3a4b391aacb9b24c']
+
+python = 'Python'
+pyver = '2.7.13'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
```

Comment:
@ebirn We have a cleaner way of doing this now, using the `%(pyver)s` and `%(pyshortver)s` template values (see also http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#avail-easyconfig-templates)

* use `-Python-%(pyver)s` as `versionsuffix` (and put it right below the `version =` line
* include `('Python', '2.7.13')` directly in `dependencies`
* remove the local variables `python`, `pyver` and `pyshortver`